app/services/grant_submission_form_services/new.rb

```diff
@@ -29,6 +29,8 @@ def self.call(grant:, user:)
                                         is_mandatory: true,
                                         response_type: 'file_upload')
         end
+      rescue ActiveRecord::RecordInvalid => invalid
+        raise ServiceError(err: invalid)
```

Comment:
this should be changed to be consistent with other usage:
            raise ServiceError.new(invalid: invalid)